pyproject.toml

```diff
@@ -18,33 +18,22 @@ python-dotenv = "^1.0.0"
 langchain-community = "^0.3.11"
 langchain-experimental = "^0.3.3"
 paramiko = "^3.4.0"
-duckduckgo-search = "^6.4.1"
-langchain-nomic = "^0.1.4"
-beautifulsoup4 = "^4.12.3"
-scikit-learn = "^1.6.0"
-nomic = {extras = ["local"], version = "^3.3.4"}
-tiktoken = "^0.8.0"
 langchain-anthropic = "^0.3.0"
 chromadb = "^0.6.2"
-torch = "2.2"
 sentence-transformers = "^3.3.1"
-google-cloud-aiplatform = "^1.38.0"
 vertexai = "^0.0.1"
 numpy = "<2.0.0"
 gradio = "^5.12.0"
 pygithub = "^2.5.0"
-speechrecognition = "^3.14.1"
 starlette = "^0.45.3"
 uvicorn = "^0.34.0"
 browser-use = "^0.1.37"
 tweepy = "^4.15.0"
-
-
+psutil = "^7.0.0"
 
 [tool.poetry.group.dev.dependencies]
 pytest-playwright = "^0.6.2"
 chromadb = "^0.6.2"
-torch = "2.2"
```

Comment:
@Am64r please check these deps - I removed all of the ones I couldn't find used in the repo, though I'm not sure on how python peer dependencies work and whether some of these are actually needed.

I removed them all because the final docker image was >10GB (too big for a docker image hosted on fly.io) so I needed to trim it down by removing deps, but we could probably re-add some if needed. The main one that cannot be included is pytorch - that is like 8+GB on its own.

There might also be another way to exclude some dependencies from the docker bundle, but I'm unfamiliar with how to do that in python.